spacy_experiment.py

```diff
@@ -71,10 +72,17 @@ def main():
     medium_model = spacy.load("en_core_web_md")
 
     print("small model:")
-    evaluate_model(data, small_model)
+    evaluate_model(data, small_model, "small_model_output.csv")
     print()
     print("medium model:")
-    evaluate_model(data, medium_model)
+    evaluate_model(data, medium_model, "medium_model_output.csv")
+    print()
+
+    # train custom model
+    training_data = load_training_data("training_data.txt")
+    custom_model = train_model(training_data)
```

Comment:
Pass the model being trained into `train_model()` (see other comment on `train_spacy.py`).